model/atmosphere/diffusion/tests/diffusion_tests/test_diffusion.py

```diff
@@ -382,7 +383,8 @@ def test_run_diffusion_single_step(
     config = construct_diffusion_config(experiment, ndyn_substeps)
     additional_parameters = diffusion.DiffusionParams(config)
 
-    diffusion_granule = diffusion_instance  # the fixture makes sure that the orchestrator cache is cleared properly between pytest runs -if applicable-
+    exchange = definitions.create_exchange(processor_props, decomposition_info)
```

Comment:
the `processor_props,decomposition_info` are not needed. Because for these tests we do not need to pass an exchange object, i.e. the default one is ok.

Reply:
Makes sense